app/models/vote.rb

```diff
@@ -0,0 +1,9 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :voteable, polymorphic: true
+
+  def voted?
+    result = where("user_id=#{current_user.id}")
+    result.count == 1
+  end
```

Comment:
why not 

where("user_id=#{current_user.id}").count == 1

we need this result variable?